### ISSUE TITLE
Disable rules for aXe

### ISSUE THREAD

**Issue by angela-choo** [angular/protractor-accessibility-plugin#14]
I'm trying to disable rules inside protractor.conf.js for aXe but does not seem to work.  Please see my code below.   Thank you!

```
plugins: [{
        axe: ({
            rules:{
                "aria-required-children": { "enabled": false },
                "color-contrast": { "enabled": false }
            }
        }),
        package: 'protractor-accessibility-plugin'
    }],
```

Also raised question here: https://github.com/dequelabs/axe-core/issues/267

**Comment by sjelin** [comment]
It appears that we're not passing on the information in the `axe` property to .  Would you be interested in writing this feature yourself?  I think you'd just want to change [this part](https://github.com/angular/protractor-accessibility-plugin/blob/master/index.js#L249) to:

```js
  AxeBuilder(browser.driver)
    .options(context.axe.options)
    .analyze(function(results) {
      deferred.resolve(results);
    });
```

**Comment by jamsea** [comment]
@angela-choo and @sjelin I started working on this functionality here: https://github.com/nfl/protractor-accessibility-plugin/tree/feature/configAXE I've been pulled off this though and won't have time to get back to it until the new year. This might help you out

**Comment by angela-choo** [comment]
@jamsea Great, I will take a look at it! 

@sjelin Sure, would be happy to work on this feature.

**Comment by angela-choo** [comment]
@sjelin added the feature and submitted PR here: angular/protractor-accessibility-plugin#15 I am wondering once the PR is approved, when will be the next release?  Thanks!

**Comment by sjelin** [comment]
I can do a release whenever

**Comment by sjelin** [comment]
0.3.0 released!  Thanks for the PR

**Comment by rolandjitsu** [comment]
Note that if you provide both:
```js
axe: {
    runOnly: {},
    rules: {}
}
```

`rules` is ignored.
